packages/aws-rfdk/lib/deadline/test/version-query.test.ts

```diff
@@ -57,44 +55,6 @@ test('VersionQuery constructor full version', () => {
     ],
   }));
   expectCDK(stack).to(haveResourceLike('AWS::Lambda::Function', {
-    Code: {
```

Comment:
How come all of this needed to get deleted?

Reply:
They should only be changed when the file(s) in its lambda change. Is it changing, randomly, without changing those files?

Reply:
Yes, I have not changed lambda in my MR and have to adjust these constants for make test pass.